pkg/cmd/cli/version/version.go

```diff
@@ -69,6 +69,7 @@ func printVersion(w io.Writer, clientOnly bool, kbClient kbclient.Client, server
 	fmt.Fprintln(w, "Client:")
 	fmt.Fprintf(w, "\tVersion: %s\n", buildinfo.Version)
 	fmt.Fprintf(w, "\tGit commit: %s\n", buildinfo.FormattedGitSHA())
+	fmt.Fprintf(w, "\tGo version: %s\n", buildinfo.GoVersion())
```

Comment:
Is this clear enough that it is not user's local go? do we want to say `Built by Go Version` instead?